### PR TITLE
use GitHub URL for node-toobusy to avoid npm install git failures

### PR DESCRIPTION
**Pull request by wenxiang**
@timotheeg @dineshsaravanan 

Tested locally, build works with node 4.x

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "commander": "~2.8.1",
     "q": "~1.4.1",
     "lodash": "~3.10.1",
-    "toobusy-js": "git+ssh://git@github.com:STRML/node-toobusy.git#3199f38dc3b"
+    "toobusy-js": "STRML/node-toobusy#v0.4.3"
```
Comment (timotheeg) [inline package.json]: In this case, toobusy 0.4.3 has been released on npm, so you can just:
```json
"toobusy-js": "0.4.3"
```

Comment (wenxiang) [inline package.json]: lol whatever, feel free to change it, i just want my fucking build to work

**Comment by timotheeg** [comment]
:+1:

**Comment by mervync** [comment]
@timotheeg just curious, is there a reason why we referred to a specific commit instead of a version?

**Comment by timotheeg** [comment]
Yes, toobusy-js had an issue we needed fixed. I made a PR upstream for it and it had been merged right away, but the npm module was not updated. I thought the maintainer may have had other plan for additional changes so I didn't make noise. 

Yesterday, when WX was having issue, I pinged the maintainer and asked if he could update the npm module, he said he had just forgotten to do it the previous time round, and he updated it within the next 5mns :expressionless:

**Comment by mervync** [comment]
ooh okay thanks for the detailed story :bow:

**Pull request merged**